docs/framework/windows-workflow-foundation/workflow-execution-properties.md

```diff
@@ -21,7 +21,7 @@ Through thread local storage (TLS), the CLR maintains an execution context for e
  Workflow execution properties provide a mechanism to add context specific properties to an activity’s environment. This allows an activity to declare which properties are in scope for its sub-tree and also provides hooks for setting up and tearing down TLS to properly interoperate with CLR objects.  
   
 ## Creating and Using Workflow Execution Properties  
- Workflow execution properties usually implement the <xref:System.Activities.IExecutionProperty> interface, though properties focused on messaging may implement <xref: System.ServiceModel.Activities.ISendMessageCallback> System.ServiceModel.Activities.ISendMessageCallback?qualifyHint=False&autoUpgrade=True and <xref: System.ServiceModel.Activities.IReceiveMessageCallback> System.ServiceModel.Activities.IReceiveMessageCallback?qualifyHint=False&autoUpgrade=True instead. To create a workflow execution property, create a class that implements the <xref:System.Activities.IExecutionProperty> interface and implement the members <xref:System.Activities.IExecutionProperty.SetupWorkflowThread%2A> and <xref:System.Activities.IExecutionProperty.CleanupWorkflowThread%2A>. These members provide the execution property with an opportunity to properly set up and tear down the thread local storage during each pulse of work of the activity that contains the property, including any child activities. In this example, a `ConsoleColorProperty` is created that sets the `Console.ForegroundColor`.  
+ Workflow execution properties usually implement the <xref:System.Activities.IExecutionProperty> interface, though properties focused on messaging may implement <xref: System.ServiceModel.Activities.ISendMessageCallback> and <xref: System.ServiceModel.Activities.IReceiveMessageCallback> instead. To create a workflow execution property, create a class that implements the <xref:System.Activities.IExecutionProperty> interface and implement the members <xref:System.Activities.IExecutionProperty.SetupWorkflowThread%2A> and <xref:System.Activities.IExecutionProperty.CleanupWorkflowThread%2A>. These members provide the execution property with an opportunity to properly set up and tear down the thread local storage during each pulse of work of the activity that contains the property, including any child activities. In this example, a `ConsoleColorProperty` is created that sets the `Console.ForegroundColor`.  
```

Comment:
broken links. there's a space between xref: and the UID. might be the root cause